tests/marginModels/test_pointSet_margin_model.xml

```diff
@@ -0,0 +1,68 @@
+<Simulation verbosity="debug">
+  <RunInfo>
+    <WorkingDir>PointSetModel</WorkingDir>
+    <Sequence>MCRun,print</Sequence>
+  </RunInfo>
+  
+  <Models>
+    <ExternalModel name="PointSetMargin" subType="SR2ML.MarginModel">
+      <variables>actualTime,marginPS1</variables>
```

Comment:
is marginPS1 a input variable, or output variable?

Reply:
marginPS1 is an output variable